e2e/playwright/point-click.spec.ts

```diff
@@ -1691,72 +1726,6 @@ sketch002 = startSketchOn(plane001)
     })
   })
 
-  // TODO: merge with above test. Right now we're not able to delete a loft
-  // right after creation via selection for some reason, so we go with a new instance
```

Comment:
The 'above' test now includes a delete step and works just fine. There's also a delete step in the offset point-and-click test, so this is now fully redundant and should be removed.